stub/src/main/java/io/grpc/stub/StreamObserver.java

```diff
@@ -68,6 +68,12 @@
    * exception is thrown by an implementation of {@code onError} no further calls to any method are
    * allowed.
    *
+   * <p>{@code t} should be a {@link io.grpc.StatusException} or {@link
+   * io.grpc.StatusRuntimeException}, but other {@code Throwable} types are possible. Callers should
```

Comment:
Do we guarantee that gRPC library always emit Status*Exception? In other words, are we making this exception only for application to pass a Throwable to StreamObserver?